src/pages/preferences/preferencesTabs/PublicKeysTab.tsx

```diff
@@ -54,19 +64,27 @@ export const PublicKeysTab: React.FC = () => {
     },
```

Comment:
Couldnt comment above but lines 38-43 i have the same comment about not manually updating the cache and instead using refetchQueries to update this data.